cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -1174,6 +1216,19 @@
                     else:
                         query_new_status = 'submitted'
                         job.set_submitted()
+                        mail_sending_job_submitted = self.app.config.get('conf').mail_sending_job_submitted
```

Comment:
I think this might be in the token too. Users may want to configure this possibility

Reply:
Will be included also in the token, along with all the email sending related parameters